yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -30,11 +30,13 @@ func init() {
 	importDataCmd.AddCommand(importDataStatusCmd)
 }
 
+//totalCount stores the total number of rows in case of import data command and the total size of the file in bytes in case of import data file command.
+//importedCount stores the number of rows imported in case of import data command and the size of the file imported in bytes in case of import data file command.
```

Comment:
Nit:
`totalCount` and `importedCount` store row-count for `import data` command and byte-count for `import data file` command.